README.md

```diff
@@ -6,6 +6,10 @@ http://www.gnu.org/licenses/lgpl.html
 -->
 
 # The Security Assertion Markup Language (SAML) Conformance Test Kit (CTK)
+
+[![Download the latest release at https://github.com/codice/saml-conformance/releases/tag/saml-ctk-1.0](https://img.shields.io/badge/Download-1.0-green.svg)](http://artifacts.codice.org/content/repositories/releases/org/codice/samlconf/samlconf/1.0/)
```

Comment:
❓ We have to remember to update the readme for the version number on every release?